lib/SIL/Utils/OwnershipUtils.cpp

```diff
@@ -225,6 +223,12 @@ bool swift::findInnerTransitiveGuaranteedUses(
         // Do not include transitive uses with 'none' ownership
         if (result->getOwnershipKind() == OwnershipKind::None)
           return true;
+        auto *phi = dyn_cast<SILPhiArgument>(result);
+        if (phi && phi->isPhi()) {
```

Comment:
fyi, I prefer either:
```
if (auto *phi = SILArgument::isPhi(value)
```

or 
```
if (auto phi = PhiValue(value))
```

We need to stop using the SILPhiArgument type and delete it ASAP

Reply:
Sure.